code/modules/mining/satchel_ore_boxdm.dm

```diff
@@ -13,7 +13,7 @@
 	if (istype(W, /obj/item/stack/ore))
 		user.transferItemToLoc(W, src)
 	else if(W.atom_storage)
-		W.atom_storage.remove_type(/obj/item/stack/ore, src)
+		W.atom_storage.remove_type(/obj/item/stack/ore, src, INFINITY, TRUE, FALSE, user, null)
```

Comment:
what is the reasoning behind these changes?

Reply:
@optimumtact as stated above, that argument is responsible for.. er.. the optimized transfer (removal) of items, and since it simply did not work before, I felt it necessary to add this to the rest of the procedures.